### ISSUE TITLE
Catch and Transmission QNAP

### ISSUE THREAD

**Issue by crashtestoz** [mipstian/catch#129]
Not that maglinks are working with Catch, can anyone suggest to to configure catch to work with Transmission running on QNAP?

Currently with maglinks enabled, Catch will just save a weblog file in the Watch folder and nothing happens.

**Comment by mipstian** [comment]
If Catch supported transmission-rpc, would that solve you problem? I personally use a small app I've written to bounce magnet links over to the machine where transmission-daemon is running. It communicates over transmission-rpc and it seems to work well. I was thinking of bringing that code into Catch directly, maybe starting as a "secret" (i.e. command line enabled) setting.

**Comment by balthisar** [comment]
> If Catch supported transmission-rpc, would that solve you problem?

Yeah, it would solve the problem for me. The karmorra mirror isn't working properly right now, so I've ended up with a bunch of webloc files, too. I can manually C&P the magnets into Transmission's web interface, but that's not really fun.

I'll look at writing a script myself in the mean time, but having this be part of Catch would be awesome.

**Comment by Pazu** [comment]
@mipstian, did transmission-rpc support happen to make it into your alpha 2 back in December?

**Comment by mipstian** [comment]
@Pazu nope, I haven’t started yet.

**Comment by RoarSlot** [comment]
@mipstian 
Any approximate timeline for this?

**Comment by mipstian** [comment]
See #163

**Comment by mipstian** [comment]
Catch 2.0 has an option to download feed contents using a script instead of the default torrent client. Not a perfect solution, but it definitely enables this use case. If someone writes a script that routes downloads to transmission-rpc, maybe share it here!
